internal/glrepo/resolver.go

```diff
@@ -66,7 +67,7 @@ func (r *ResolvedRemotes) BaseRepo(prompt bool) (Interface, error) {
 	for _, r := range r.remotes {
 		if r.Resolved == "base" {
 			return r, nil
-		} else if r.Resolved != "" {
+		} else if strings.HasPrefix(r.Resolved, "base:") {
```

Comment:
same here, we need to distinguish resolutions made for `head:` and for `base:`

Reply:
Oh okay. I get it